File: C++/TemperatureConverter.cpp
#include <iostream>
#include <cmath>
#include <windows.h>

using namespace std;
int main()
{ 
 system("CLS");
 system("Color 0A");  
 int choice;
 float value,convert;
 cout<<"Please select your choice:\n1. Celcius to Fahreinheit\n2. Fahreinheit to Celcius\n3. Celcius to Kelvin\n4. Kelvin to Celcius\n5. Fahreinheit to Kelvin\n6. Kelvin to Fahreinheit\n";
 cin>>choice;
 switch(choice) {
   case 1: cout<<"Please enter the Temperature in Celcius:";
           cin>>value;
           convert=(value * 1.8) + 32;
           cout<<"Celcius to Fahreinheit: "<<value <<"°C = " <<convert <<"°F.";
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);
   break;
   case 2: cout<<"Please enter the Temperature in Fahreinheit:";
           cin>>value;
           convert=(value - 32) * 0.5556;
           cout<<"Fahreinheit to Celcius: "<<value <<"°F = " <<convert <<"°C.";   
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);   
   break;
   case 3: cout<<"Please enter the Temperature in Celcius:";
           cin>>value;
           convert=value+273;
           cout<<"Celcius to Kelvin: "<<value <<"°C = " <<convert <<"°K.";   
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);   
   break;
   case 4: cout<<"Please enter the Temperature in Kelvin:";
           cin>>value;
           convert=value-273;
           cout<<"Kelvin to Celcius: "<<value <<"°K = " <<convert <<"°C.";   
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);   
   break;
   case 5: cout<<"Please enter the Temperature in Fahreinheit:";
           cin>>value;
           convert=((value - 32) * 0.5556) + 273;
           cout<<"Celcius to Fahreinheit: "<<value <<"°C = " <<convert <<"°F.";   
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);   
   break;
   case 6: cout<<"Please enter the Temperature in Kelvin:";
           cin>>value;
           convert=((value - 273) * 1.8) + 32;
           cout<<"Celcius to Fahreinheit: "<<value <<"°C = " <<convert <<"°F.";   
           cout<<"\nThanks for using Temperature Converter.";
           exit(1);   
   break;
   default: cout<<"Please choose a valid option.";
   break; 
 }
} 

File: Python/TemperatureConverter.py
import colorama
from colorama import Fore

print("Welcome to Temperature Convertor!")
valid = False
while not valid:
    try:
        x = int(input(Fore.GREEN + "Select what you want to do:\nEnter 1 to Convert Celcius to Fahrenheit\nEnter 2 to Convert Fahrenheit to Celcius\nEnter 3 to Convert Celcius to Kelvin\nEnter 4 to Convert Kelvin to Celcius\nEnter 5 to Convert Fahrenheit to Kelvin\nEnter 6 to Convert Kelvin to Fahrenheit"))
        valid = True
    except ValueError:
        print("Valid option not inputted. Retrying the program.")
question = int(input(Fore.MAGENTA + "Enter the value to Convert: "))
if x == 1:
    ans = (question * 1.8) + 32
    print(Fore.CYAN + str(question) + "°C is " + str(ans) + "°F.") 
    print(Fore.RED + "Thanks for using Temperature Convertor.")
    exit()
if x == 2:
    ans = (question - 32) * 0.5556
    print(Fore.CYAN + str(question) + "°F is " + str(ans) + "°C.")
    print(Fore.RED + "Thanks for using Temperature Convertor.") 
    exit()    
if x == 3:
    ans = (question + 273.15)
    print(Fore.CYAN + str(question) + "°C is " + str(ans) + "°K.")
    print(Fore.RED + "Thanks for using Temperature Convertor.")
    exit()    
if x == 4:
    ans = (question - 273.15)
    print(Fore.CYAN + str(question) + "°K is " + str(ans) + "°C.")
    print(Fore.RED + "Thanks for using Temperature Convertor.")
    exit()    
if x == 5:
    ans = ((question - 32) * 0.5556) + 273.15
    print(Fore.CYAN + str(question) + "°F is " + str(ans) + "°K.")
    print(Fore.RED + "Thanks for using Temperature Convertor.")
    exit()    
if x == 6:
    ans = ((question - 273.15) * 1.8) + 32
    print(Fore.CYAN + str(question) + "°K is " + str(ans) + "°F.")
    print(Fore.RED + "Thanks for using Temperature Convertor.")
    exit()   